internal/controllers/router/tests/import/01-create-trap-router.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Please add a comment explaining the reason why we have this resource.
You can copy what I have added [here](https://github.com/k-orc/openstack-resource-controller/pull/278/files#diff-d88dd589a2e0658bb73e69d8a37045eff06d946f7f8171d13ef33dd371da0584R2-R4).